sonic-xcvrd/scripts/xcvrd

```diff
@@ -432,16 +444,23 @@ def del_port_sfp_dom_info_from_db(logical_port_name, int_tbl, dom_tbl):
             sys.exit(NOT_IMPLEMENTED_ERROR)
 
 # recover missing sfp table entries if any
-def recover_missing_sfp_table_entries(sfp_util, int_tbl, status_tbl, stop_event):
+def recover_missing_sfp_table_entries(self, sfp_util, stop_event):
```

Comment:
Since this isn't a method of any class, the first parameter shouldn't be named `self`, *OR* it should be moved to become a class method.

Reply:
Thanks Joe, I have reverted back to use int_tbl, status_tbl itself. The int_tbl, status_tbl are now list datatype and indexed by the asic_id. In case of single ASIC patform the asic_id is 0 , ( returned by the get_asic_id_from_namespace() API in MultiAsic class in daemon_base.py )